src/components/TopicInput.tsx

```diff
@@ -73,6 +73,7 @@ const SearchInput: React.FC<{
   return (
     <Select
```

Comment:
Can we set [maxTagCount](https://ant.design/components/select#select-demo-responsive) to prevent changing the height of select?

<img width="1512" alt="image" src="https://github.com/AttackOnMorty/github-ranking/assets/37203901/a2545ddc-c177-4494-a7be-7ddec15ad991">
Figure: Select height changed